pkg/environment/environment.go

```diff
@@ -134,6 +146,37 @@ func GetModulesRole() string {
 	return os.Getenv(VaultModulesRoleKey)
 }
 
+// GetResourcesPollingInterval returns the time interval to check the
+// status of the resources deployed by the manager. The interval is specified
+// in milliseconds.
+func GetResourcesPollingInterval() (time.Duration, error) {
+	intervalStr := os.Getenv(ResourcesPollingInterval)
+	if intervalStr == "" {
+		return defaultPollingInterval, nil
+	}
+	interval, err := strconv.Atoi(intervalStr)
+	if err != nil {
+		return defaultPollingInterval, err
```

Comment:
very strange, you return default value and error, it should be one of them

Reply:
done. also for  `GetDataPathMaxSize` function which uses the same pattern.